script/build.sh

```diff
@@ -106,9 +106,16 @@ function docker_cmd()
         echo "building ${DOCKERHUB_REGISTRY}:${HASH_TAG}..."
         docker build -t $DOCKERHUB_REGISTRY:${HASH_TAG} .
 
+        echo "scanning docker image..."
+        docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v "$HOME"/Library/Caches:/root/.cache/ aquasec/trivy --exit-code 1 --severity CRITICAL "$DOCKERHUB_REGISTRY":"${HASH_TAG}"
+
+
         echo "building ${DOCKERHUB_TESTS_REGISTRY}:${HASH_TAG}..."
         cd automation
         docker build -f Dockerfile-tests -t $DOCKERHUB_TESTS_REGISTRY:${HASH_TAG} .
+
+        echo "scanning docker image..."
+        docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v "$HOME"/Library/Caches:/root/.cache/ aquasec/trivy --exit-code 1 --severity CRITICAL "$DOCKERHUB_TESTS_REGISTRY":"${HASH_TAG}"
```

Comment:
@zbedo does `"$HOME"/Library/Caches` work for non-MacOS systems such as Jenkins, Travis, Circle?